src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,11 @@
+@import "../../colors";
+@import "../../typography";
+
+.view {
+    .guidelines-footer {
+        text-align: center;
```

Comment:
This is what the HTML produces in my browser:
![screenshot](https://cloud.githubusercontent.com/assets/12952370/14887166/8fbb1dc6-0d22-11e6-8254-67106b203f49.png)
It's kind of odd, images don't normally align with the `text-align` rule. But this is what my CSS for the footer is:
![screenshot](https://cloud.githubusercontent.com/assets/12952370/14887254/ebf1ae16-0d22-11e6-96c0-72274f12092d.png)